lib/cucumber/platform.rb

```diff
@@ -14,6 +14,7 @@ module Cucumber
     RUBY_2_2      = RUBY_VERSION =~ /^2\.2/
     RUBY_2_1      = RUBY_VERSION =~ /^2\.1/
     RUBY_2_3      = RUBY_VERSION =~ /^2\.3/
+    RUBY_2_4      = RUBY_VERSION =~ /^2\.4/
 
```

Comment:
> this PR removes the only usage,

I am still not sure above point. actually this PR can remove these constants?
or this PR should not touch these constants?

The constants RUBY_2_* are used in `gem_tasks/cucumber.rake`.

```
$ grep -r RUBY_2_ *
gem_tasks/cucumber.rake:                   elsif Cucumber::RUBY_2_0
gem_tasks/cucumber.rake:                   elsif Cucumber::RUBY_2_1
lib/cucumber/runtime.rb:    include FixRuby21Bug9285 if Cucumber::RUBY_2_1 || Cucumber::RUBY_2_2 || Cucumber::RUBY_2_3
lib/cucumber/platform.rb:    RUBY_2_2      = RUBY_VERSION =~ /^2\.2/
lib/cucumber/platform.rb:    RUBY_2_1      = RUBY_VERSION =~ /^2\.1/
lib/cucumber/platform.rb:    RUBY_2_3      = RUBY_VERSION =~ /^2\.3/
```